app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::V1::AuthenticationController < ApplicationController
+    before_action :authorize_request, except: [:login, :create]
+    before_action :get_authorize_user, only: [:retrive]
```

Comment:
the correct name has to be get_autorized_user because you are talking about a cuality of the user not about the action.

Reply:
Solved